setup.py

```diff
@@ -114,7 +114,7 @@ def find_python_files(base: Path) -> List[Path]:
     },
     test_suite="tests.test_black",
```

Comment:
Unrelatedly, isn't this wrong now? We have more test files than this.

Reply:
This is deprecated anyways and does not help pytest. So it can be removed I’m pretty sure.